File: react-frontend/src/models/Tournament.ts
import { User } from "./User";

export interface Tournament {
    id: number;
    t_name: string;
    t_country: string;
    t_start_date: string;
    t_end_date: string;
    t_type: string;
    nb_registers: number;
    added_by: User;
}

File: tennis/application/Models/Tournament.py
# from django.db import models
# from ..Models import TennisPlayer, TournamentRegistration
#
# class Tournament(models.Model):
#     t_name = models.CharField(max_length=100)
#     t_country = models.CharField(max_length=100)
#     t_start_date = models.DateField()
#     t_end_date = models.DateField()
#     t_type = models.CharField(max_length=100)
#     players = models.ManyToManyField(TennisPlayer, through='TournamentRegistration')
#
#     def __str__(self):
#         return self.t_name